infra/pre/base/kustomization.yaml

```diff
@@ -8,5 +8,5 @@ resources:
   - ../../../core/app-config-service
   - ../../../core/grafana
   # - ../../../core/calico
```

Comment:
merge with main for calico to be re-enabled ?

Reply:
Done. Pulled the changes from main.